Models/login.js

```diff
@@ -0,0 +1,42 @@
+// const mongoose = require("mongoose");
+
+// const login_schema = mongoose.Schema({
+// 	username: { type: String, required: true },
+// 	location: { type: String, required: true },
+// });
+
+// module.exports = mongoose.model("login", login_schema); /// login collections created
+
+const mongoose = require("mongoose");
+const postSchema = new mongoose.Schema({
+	username: {
+		type: String,
+		required: true,
+	},
+
+	location: {
+		type: String,
+		required: true,
+	},
+
+	comment: {
+		////newly added fn
```

Comment:
Comments don't seem in line with the code. Can you please remove the unnecessary ones?